docs/getting-started-guide.md

```diff
@@ -0,0 +1,168 @@
+# How to set up and run the Co-Simulation Demo
```

Comment:
There are a few demos in this repository, maybe call out the specific one you're show casing.  Then add a ditta/dot diagram(whatever github is most happiest with) that shows how all the parts of the demo come together.  Below that you can then note what tools and purposes each part has.
By adding that visual and outline, the rest of this setup/execute sequence should be easier to follow.  You may even want to breakout the steps and elaborate as you go.

Reply:
This is a great idea since many people would be able to better visualize what is happening. We are working on some other basic diagrams so we could possibly repurpose those